Holovibes/includes/api/contrast_api.hh

```diff
@@ -6,351 +6,355 @@
  */
 #pragma once
 
-#include "API.hh"
 #include "enum_window_kind.hh"
+#include "common_api.hh"
 
 namespace holovibes::api
 {
 
-#pragma region Internals
+class ContrastApi : public IApi
+{
 
-WindowKind get_current_window_type(); // Forward declaration
+  public:
+    ContrastApi(const Api* api)
+        : IApi(api)
+    {
+    }
+#pragma region Internals
 
-/*! \brief Returns the setting of the window kind specified
- *
- * \param[in] kind the window kind
- * \return ViewWindow the setting of the window kind
- */
-inline ViewWindow get_window(WindowKind kind)
-{
-    switch (kind)
+    /*! \brief Returns the setting of the window kind specified
+     *
+     * \param[in] kind the window kind
+     * \return ViewWindow the setting of the window kind
+     */
+    inline ViewWindow get_window(WindowKind kind) const
     {
-    case WindowKind::XYview:
-        return GET_SETTING(XY);
-    case WindowKind::XZview:
-        return GET_SETTING(XZ);
-    case WindowKind::YZview:
-        return GET_SETTING(YZ);
-    default:
-        return GET_SETTING(Filter2d);
+        switch (kind)
+        {
+        case WindowKind::XYview:
+            return GET_SETTING(XY);
+        case WindowKind::XZview:
+            return GET_SETTING(XZ);
+        case WindowKind::YZview:
+            return GET_SETTING(YZ);
+        default:
+            return GET_SETTING(Filter2d);
+        }
     }
-}
 
 #pragma endregion
```

Comment:
Maybe put internals as private members, so that they're not callable
Is also valid for other internals in other API classes